service_listener.go

```diff
@@ -75,16 +100,11 @@ func main() {
 		glog.Infof("Found %d DNS services in all namespaces with selector %q", len(services.Items), selector)
 		for i := range services.Items {
 			s := &services.Items[i]
-			ingress := s.Status.LoadBalancer.Ingress
-			if len(ingress) < 1 {
-				glog.Warningf("No ingress defined for ELB")
-				continue
-			}
-			if len(ingress) < 1 {
-				glog.Warningf("Multiple ingress points found for ELB, not supported")
+			hn, err := serviceHostname(s)
+			if err != nil {
+				glog.Warningf("Couldn't find hostname: %s", err)
```

Comment:
Ok, beginning to think this is intentional. Whats the difference between `[Warning|Fata|]f` and just `[Warning|Fatal]`?